packages/preview/whalogen-0.1.0/typst.toml

```
@@ -0,0 +1,11 @@
+[package]
+name = "whalogen"
+version = "0.1.0"
+repository = "https://github.com/schang412/whalogen"
+entrypoint = "lib.typ"
+authors = [
+    "Spencer Chang <spencer@sycee.xyz>"
+]
+license = "Apache-2.0"
+description = "Typsetting chemical formulas, a port of mhchem"
```

```suggestion
description = "Typesetting chemical formulas, a port of mhchem"
```
not sure whether "typsetting" was intentional :D